.github/workflows/docs.yml

```diff
@@ -1,39 +0,0 @@
-name: Documentation linter
-
-on:
-  pull_request:
-    paths:
-      - 'docs/**'
-      - '*.md'
-      - '.github/workflows/docs.yml'
-
-  push:
-    branches:
-      - main
-    paths:
-      - 'docs/**'
-      - '*.md'
-      - '.github/workflows/docs.yml'
-
-jobs:
-  check-all-files:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: documentation quality check
-        uses: errata-ai/vale-action@v1.4.0
-        with:
-          files: __onlyModified
-          onlyAnnotateModifiedLines: true
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Broken link checker
-        uses: gaurav-nelson/github-action-markdown-link-check@v1
-        with:
-          config-file: '.github/workflows/markdown.links.config.json'
-          folder-path: 'docs/'
-          check-modified-files-only: 'yes'
-          file-extension: '.md'
-          base-branch: 'main'
```

Comment:
Can we possibly keep it around or replace it as a pre-commit hook?

Reply:
I would suggest having this as a Cron job then, as they did in [here](https://github.com/GoogleChrome/lighthouse/blob/master/.github/workflows/cron-weekly.yml)

Reply:
I can see a benefit of it being run as a cronjob. I'm not sure about the rest of the team, but I miss to run into issues with this particular link checker on a regular basis. Perhaps finding another one altogether would also be useful...

Reply:
Perhaps we will save the link checker discussion for another time.